src/later.cpp

```diff
@@ -81,6 +81,17 @@ void execLater(Rcpp::Function callback, double delaySecs) {
   doExecLater(callback, delaySecs);
 }
 
+// [[Rcpp::export]]
+double secsToNext() {
+  Optional<Timestamp> nextTime = callbackRegistry.nextTimestamp();
+  if (!nextTime.has_value()) {
+    return R_PosInf;
+  } else {
+    Timestamp now;
+    return (*nextTime).diff_secs(now);
```

Comment:
Is this syntax needed instead of `->` because of the way the Optional class works?

Reply:
Yeah. I implemented the `*` operator but not the `->` operator--I probably could/should.